### PR TITLE
chore: add project and idea to json

### PR DESCRIPTION
**Pull request by NikkiHmltn**

### CODE REVIEW

File: data/ideas.json [hunk 1]
```diff
@@ -34,5 +34,11 @@
         "description": "This repository is about the code to extract Convolutional-neural-network layers and visualise the filters and featuremaps",
         "level": "basic",
         "tags": "feature-maps-and-filter-visualizer, machine-learning"
+    },
+    {
+        "title": "Rock, Paper, Scissors",
+        "description": "Create a rock paper scissors game! Try recreating the classic game, or put your own fun twist on it.",
+        "level": "beginner",
```
Comment (ming-tsai) [inline data/ideas.json:41]: The level only support basic, intermediate and advance
```suggestion
        "level": "basic",
```

File: data/projects.json [hunk 2]
```diff
@@ -138,6 +138,16 @@
...
         "site": "",
         "tags": "car-game,c,c++,graphics-project"
+    },
+    {
+        "name": "Tamagotchi-Game",
+        "display": "Tamagotchi Game",
+        "description": "A game where you manage a creature with 3 needs (usually eat, sleep, entertainment) and as your creature ages, it morphs.",
+        "author": "NikkiHmltn",
+        "category": "game",
+        "level": "beginner",
```
Comment (ming-tsai) [inline data/projects.json:148]: The level only support basic, intermediate, and advance
```suggestion
        "level": "basic",
```

File: data/projects.json [hunk 3]
```diff
@@ -138,6 +138,16 @@
         "level": "basic",
         "site": "",
         "tags": "car-game,c,c++,graphics-project"
+    },
+    {
+        "name": "Tamagotchi-Game",
```
Comment (ming-tsai) [inline data/projects.json:143]: The repo name doesn't exist, please check the repo name or its visibility
image

File: data/ideas.json [hunk 4]
```diff
@@ -34,5 +34,11 @@
...
+        "tags": "game, classic, rock-paper-scissors"
```
Comment (ming-tsai) [inline data/ideas.json:42]: The first tag are using to match the repo, so please move `rock-paper-scissors` to first tag
```suggestion
        "tags": "rock-paper-scissors, game, classic"
```

**Review by ming-tsai — CHANGES_REQUESTED** [review]
Hi @NikkiHmltn,
I added a couple of comments.

File: data/projects.json (hunk 2 above)
Comment (NikkiHmltn) [reply data/projects.json]: I made the requested changes to the idea. Thank you for the feedback!

File: data/projects.json (hunk 3 above)
Comment (NikkiHmltn) [reply data/projects.json]: I haven't finished the game yet and it's private. I removed the project and will come back and add it when it's finished. Thank you!

**Review by ming-tsai — APPROVED** [review]

**Pull request merged**